packages/sdk/package.json

```diff
@@ -60,6 +60,7 @@
     "mocha": "^8.4.0",
     "nyc": "^15.1.0",
     "prettier": "^2.3.1",
+    "sinon": "^13.0.2",
```

Comment:
Tabs vs spaces

Reply:
I actually thought that you needed sinon to use mocha since I've pretty much only ever used jest.

Reply:
> > > This PR changes implementation code, but doesn't include a changeset. Did you forget to add one?
> > 
> > 
> > @roninjin10 Can you add a changeset?
> 
> He did smile

Whoops sorry thought I didn't see one

Reply:
Wait so are you using `sinon`? If not, can you delete it as a dep

Reply:
Ok i see it being used, disregard this comment